### PR TITLE
[onert] Fix const tensors to use ExternalData on cpu

### PR DESCRIPTION
**Pull request by YongseopKim**
- ExternalTensor: get Data from external backend
- IConstantInitializer::registerExternalInitializer: set tensor to use a
data from external
- StaticTensorManager's const mem mgr does nothing

ONE-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
Inv3

before
```
odroid@odroid:/home/dragon/Works/github/ONE$ BACKENDS="cpu" ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/inception_v3/ -w5 -r10 -m0
===================================
MODEL_LOAD   takes 34.421 ms
PREPARE      takes 1171.124 ms
EXECUTE      takes 1594.680 ms
- MEAN     :  1594.680 ms
- MAX      :  1626.545 ms
- MIN      :  1539.631 ms
- GEOMEAN  :  1594.489 ms
===================================


odroid@odroid:/home/dragon/Works/github/ONE$ BACKENDS="cpu" ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/inception_v3/ -w5 -r1 -m1
===================================
MODEL_LOAD   takes 34.673 ms
PREPARE      takes 1120.102 ms
EXECUTE      takes 1543.217 ms
- MEAN     :  1543.217 ms
- MAX      :  1543.217 ms
- MIN      :  1543.217 ms
- GEOMEAN  :  1543.217 ms
===================================
RSS
- MODEL_LOAD   takes 11572 kb
- PREPARE      takes 196648 kb
- EXECUTE      takes 224668 kb
- PEAK         takes 224668 kb
===================================
HWM
- MODEL_LOAD   takes 11572 kb
- PREPARE      takes 203688 kb
- EXECUTE      takes 227344 kb
- PEAK         takes 227344 kb
===================================
PSS
- MODEL_LOAD   takes 12123 kb
- PREPARE      takes 199023 kb
- EXECUTE      takes 222708 kb
- PEAK         takes 222708 kb
===================================
```

after
```
odroid@odroid:/home/dragon/Works/github/ONE$ BACKENDS="cpu" ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/inception_v3/ -w5 -r10 -m0
===================================
MODEL_LOAD   takes 35.260 ms
PREPARE      takes 1317.616 ms
EXECUTE      takes 1253.036 ms
- MEAN     :  1253.036 ms
- MAX      :  1264.918 ms
- MIN      :  1234.496 ms
- GEOMEAN  :  1253.012 ms
===================================
odroid@odroid:/home/dragon/Works/github/ONE$ BACKENDS="cpu" ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/inception_v3/ -w5 -r1 -m1
===================================
MODEL_LOAD   takes 48.472 ms
PREPARE      takes 1382.869 ms
EXECUTE      takes 1274.612 ms
- MEAN     :  1274.612 ms
- MAX      :  1274.612 ms
- MIN      :  1274.612 ms
- GEOMEAN  :  1274.612 ms
===================================
RSS
- MODEL_LOAD   takes 11572 kb
- PREPARE      takes 109968 kb
- EXECUTE      takes 136922 kb
- PEAK         takes 136922 kb
===================================
HWM
- MODEL_LOAD   takes 11572 kb
- PREPARE      takes 109968 kb
- EXECUTE      takes 138508 kb
- PEAK         takes 138508 kb
===================================
PSS
- MODEL_LOAD   takes 12239 kb
- PREPARE      takes 108007 kb
- EXECUTE      takes 136624 kb
- PEAK         takes 136624 kb
===================================
```

**Comment by YongseopKim** [comment]
Mv2

before
```
odroid@odroid:/home/dragon/Works/github/ONE$ BACKENDS="cpu" ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/mobilenet_v2_1.0_224/ -w5 -r1 -m1
===================================
MODEL_LOAD   takes 30.952 ms
PREPARE      takes 835.981 ms
EXECUTE      takes 322.207 ms
- MEAN     :  322.207 ms
- MAX      :  322.207 ms
- MIN      :  322.207 ms
- GEOMEAN  :  322.207 ms
===================================
RSS
- MODEL_LOAD   takes 7384 kb
- PREPARE      takes 35964 kb
- EXECUTE      takes 44239 kb
- PEAK         takes 44239 kb
===================================
HWM
- MODEL_LOAD   takes 7384 kb
- PREPARE      takes 35964 kb
- EXECUTE      takes 44943 kb
- PEAK         takes 44943 kb
===================================
PSS
- MODEL_LOAD   takes 5522 kb
- PREPARE      takes 37794 kb
- EXECUTE      takes 43575 kb
- PEAK         takes 43575 kb
===================================
```

after
```
odroid@odroid:/home/dragon/Works/github/ONE$ BACKENDS="cpu" ./Product/out/bin/nnpackage_run ./benchmark_nnpkg_models/mobilenet_v2_1.0_224/ -w5 -r1 -m1
===================================
MODEL_LOAD   takes 31.274 ms
PREPARE      takes 1350.377 ms
EXECUTE      takes 303.850 ms
- MEAN     :  303.850 ms
- MAX      :  303.850 ms
- MIN      :  303.850 ms
- GEOMEAN  :  303.850 ms
===================================
RSS
- MODEL_LOAD   takes 7624 kb
- PREPARE      takes 28240 kb
- EXECUTE      takes 31712 kb
- PEAK         takes 31712 kb
===================================
HWM
- MODEL_LOAD   takes 7624 kb
- PREPARE      takes 29592 kb
- EXECUTE      takes 32086 kb
- PEAK         takes 32086 kb
===================================
PSS
- MODEL_LOAD   takes 5745 kb
- PREPARE      takes 27365 kb
- EXECUTE      takes 30337 kb
- PEAK         takes 30337 kb
===================================
```

File: runtime/onert/core/src/backend/BackendContext.cc [hunk 1]
```diff
@@ -57,7 +58,10 @@ void BackendContext::initConsts()
     const auto &obj = _graph->operands().at(ind);
     if (obj.isConstant() && !constant_initializer->exist(ind))
     {
-      constant_initializer->registerPermuteInitializer(ind, obj);
+      if (_backend->config()->id() == "cpu")
+        constant_initializer->registerExternalInitializer(ind, obj);
+      else
+        constant_initializer->registerPermuteInitializer(ind, obj);
```
Comment (wateret) [inline runtime/onert/core/src/backend/BackendContext.cc:64]: I hope we could avoid this way.

Comment (YongseopKim) [reply runtime/onert/core/src/backend/BackendContext.cc:64]: Yes, it looks ugly. I'll try avoiding this way.

File: runtime/onert/core/src/backend/BackendContext.cc [hunk 2]
```diff
@@ -56,9 +57,7 @@ void BackendContext::initConsts()
   {
     const auto &obj = _graph->operands().at(ind);
     if (obj.isConstant() && !constant_initializer->exist(ind))
-    {
-      constant_initializer->registerPermuteInitializer(ind, obj);
-    }
+      constant_initializer->registerDefaultInitializer(ind, obj);
```
Comment (YongseopKim) [inline runtime/onert/core/src/backend/BackendContext.cc:60]: @wateret how about this? According to your comemnt, I revise with `registerDefaultInitializer`.

Comment (wateret) [reply runtime/onert/core/src/backend/BackendContext.cc:60]: Looks much better 😄

**Pull request closed without merging**